src/types/index.ts

```diff
@@ -126,8 +130,15 @@ export enum TokenIdentifierType {
   Isin = 'Isin',
   Cusip = 'Cusip',
   Cins = 'Cins',
+  Lei = 'Lei',
+}
+
+export enum KnownPortfolioKind {
+  Default = 'Default',
 }
 
+export type PortfolioKind = KnownPortfolioKind | { user: number };
```

Comment:
No need for these types. `PortfolioId` should just be an internal type, and

```
export interface PortfolioId {
  did: string;
  number?: number;
}
```

Internally you know that if `number` is undefined, it's the default portfolio. Also, note that `did` should be a `string` and not an `IdentityId` (never mix polkadot types with our own)